src/style.css

```diff
@@ -1,3 +1,86 @@
+:root {
+  --active-color: #00acdc;
+}
+
+html {
+  position: relative;
+  align-items: center;
+  height: 60px;
+  font-family: "Roboto", sans-serif;
+  --active-color: #00acdc;
```

Comment:
Be careful when creating the variables, because you already added the `--active-color` variable earlier.